jbig2/jbig2_decompress_image.go

```diff
@@ -1,28 +1,36 @@
 /*
  * This example showcases the decompression of the jbig2 encoded image and storing into
  * commonly used jpg format.
- *
- * As the input for this test the result of the compression example would be used (lossless image).
```

Comment:
is this text not relevant?

Reply:
I guess you are specifically discussing the case when first encoding, then decoding with this?
In that case I guess the text can be removed.
In our article it can be demoed.

Reply:
I've changed the description of that image example, to the one that matches current implementation more. Done.